src/main/java/com/iwaconsolti/league/service/TeamService.java

```diff
@@ -0,0 +1,67 @@
+package com.iwaconsolti.league.service;
+import com.iwaconsolti.league.Config.TeamConfig;
+import com.iwaconsolti.league.model.PlayerModel;
+import com.iwaconsolti.league.model.TeamModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class TeamService {
+    private List<TeamModel> teamslist = new ArrayList<>();
+    private final PlayerService playerService;
+    private final TeamConfig teamConfig;
+
+    @Autowired
+    public TeamService(PlayerService playerService, TeamConfig teamConfig) {
+        this.playerService = playerService;
+        this.teamConfig = teamConfig;
+        teamslist = teamConfig.getTeamsconfiglist();
+    }
+
+    public TeamModel insertTeam(TeamModel team) {
+        List<PlayerModel> players = playerService.getPlayer().stream()
```

Comment:
Why do I need to check the players when a team is created. there should not be any player created for it yet